packages/yelp_xsl.rb

```diff
@@ -0,0 +1,27 @@
+
+require 'package'
+
+class Yelp_xsl < Package
+  description 'yelp-xsl is a collection of programs and data files to help you build, maintain, and distribute documen'
```

Comment:
Still missing the last part of the description.  Should it be `documentation`?

Reply:
Please slow down and pay attention to detail. :)

Reply:
I have tested it several times.